### PR TITLE
optimize chunks

### PR DESCRIPTION
**Pull request by collin**
Starting to optimize output based on information from stats.

### CODE REVIEW

File: scripts/webpack-build-stats [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+export NODE_ENV=production
```
Comment (kand) [inline scripts/webpack-build-stats]: @daytonn will this export this variable to your shell env once this is run?

**Comment by kand** [comment]
👍

File: scripts/webpack-build-stats (hunk 1 above)
Comment (daytonn) [inline scripts/webpack-build-stats]: yes `export` sets the variable globally, we can just remove the `export` to make it local. If you want to get nuts you can do `local NODE_ENV=production`.

**Comment by daytonn** [comment]
👍

File: scripts/webpack-build-stats (hunk 1 above)
Comment (collin) [inline scripts/webpack-build-stats]: This doesn't pollute the parent process.

```
scripts/webpack-build-stats
echo $NODE_ENV 
```
Will not print out `production`

**Pull request merged**